source/common/common/utility.cc

```diff
@@ -4,12 +4,11 @@
 #include <chrono>
 #include <cmath>
 #include <cstdint>
+#include <iostream>
```

Comment:
This is not required?

Reply:
> This is not required?

Thanks for pointing this out! 
Yes, it is not required. I have removed this line. And the format is also fixed.